command/agent/auth/auth.go

```diff
@@ -4,15 +4,21 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"math"
 	"math/rand"
 	"net/http"
 	"time"
 
-	hclog "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/vault/api"
 	"github.com/hashicorp/vault/sdk/helper/jsonutil"
 )
 
+const (
+	initialBackoff    = 1 * time.Second
+	defaultMaxBackoff = 1 * time.Minute
```

Comment:
I would be inclined to go a little higher than this as the default. If baseline Vault traffic is anywhere close to server capacity, and all Vault agents are retrying every minute during an outage, that's likely to still be significantly higher traffic than baseline. WDYT?

Reply:
I agree.  Maybe 5 minutes?

Reply:
Sounds good. I updated it from 1->5m.